Revert "Fix filters in amp-bind reference (#27143)"

sebastianbenz opened the pull request:
The commit broke the amp.dev build due to unclosed filter.

caroqliu commented on extensions/amp-bind/amp-bind.md:
```diff
@@ -261,6 +261,8 @@ An `amp-state` element must contain a child `<script>` element.
 
 #### Attributes
 
+[filter formats="websites, stories, ads"]
```
It looks like this `filter` line appears both above and below the "Attributes" header with the change, the one above being unclosed.

sebastianbenz replied:
You're right - (our parser doesn't care though). Went ahead and fixed the filters.

sebastianbenz commented:
OK, looked through the doc and fixed a few things:

* wrong indentation for attributes and events (sidenote: I find this structure a bit confusing as amp-bind does not directly match to an amp-bind element so it might make more sense to move attributes/actions into the state section //cc @CrystalOnScript). 
* removed stories from filters as stories don't support amp-bind
* fixed duplicate filter instruction

@caroqliu PTAL

kristoferbaxter commented on extensions/amp-bind/amp-bind.md:
```diff
@@ -257,31 +257,34 @@ An `amp-state` element must contain a child `<script>` element.
 
+AMP batches XMLHttpRequests (XHRs) to JSON endpoints, that is, you can use a single JSON data request as a data source for multiple consumers (e.g., multiple `amp-state` elements) on an AMP page.
+
+For example, if your `amp-state` element makes an XHR to an endpoint, while the XHR is in flight, all subsequent XHRs to the same endpoint won't trigger and will instead return the results from the first XHR.
```
Nice move, much more relevant to the section than a note toward the end.

sebastianbenz commented:
Thanks for the review!
